inst/extdata/install-packages.R

```diff
@@ -0,0 +1,177 @@
+.releasePaths <- list(
+  ospsuite.utils = "https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/releases/download/v1.3.17/ospsuite.utils_1.3.17.zip",
+  tlf = "https://github.com/Open-Systems-Pharmacology/TLF-Library/releases/download/v1.4.89/tlf_1.4.89.zip",
+  ospsuite = "https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases/download/v11.0.123/ospsuite_11.0.123.zip"
+)
+# Download paths of latest develop package versions
+.developPaths <- list(
+  ospsuite.utils = "https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip",
+  tlf = "https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/tlf-library/artifacts/tlf.zip",
+  ospsuite = "https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-r/artifacts/ospsuite.zip"
+)
+
+#' Test connection to PK-Sim
+#'
+#' @param pkSimPath Path where PK-Sim is installed. If this is not specified
+#' (`NULL`), path is estimated by the `ospsuite` package.
+#'
+#' @return `TRUE` if `ospsuite::initPKSim(pkSimPath)` is successful.
+testPKSIMConnection <- function(pkSimPath = NULL) {
+  # returns TRUE if PK-Sim.R dll can be loaded,
+  # or raises errors otherwise
+  library(ospsuite)
+  ospsuite::initPKSim(pkSimPath)
+  return(TRUE)
+}
+
+# Remove functions created by this script from the environment
+cleanEnvironment <- function() {
+  rm(
+    packageInstallationMessages,
+    .releasePaths,
+    .developPaths,
+    testPKSIMConnection,
+    cleanEnvironment,
+    displayProgress,
+    installOSPPackages,
+    installPackagesGlobally,
+
+    pos = 1
+  )
+}
+
+displayProgress <- function(current, success = TRUE, message = NULL, suppressOutput = TRUE) {
+  states <- c(
+    "Installing RENV", "Installing CRAN packages",
+    "Checking RTOOLS", "Installing rClr", "Installing ospsuite.utils",
+    "Installing tlf", "Installing ospsuite",
+    "Testing PK-Sim connection",
+    "Installation successful"
+  )
+  if (suppressOutput) {
+    cat("\014")
+    for (i in seq_along(states)) {
+      cat(paste0(ifelse(i < which(states == current), "V", ifelse(i > which(states == current), ".", ifelse(success == FALSE, "X", ".")))), " ", states[[i]], "\n")
+    }
+  } else {
+    cat(paste0(current, "\n"))
+  }
+  if (!is.null(message)) {
+    message(message)
+  }
+}
+
+#' Install all osps packages and their dependencies.
+#'
+#' @param rtoolsPath Path to where rtools are installed. If `NULL` (default),
+#' the path is deduced from system environment variables.
+#' @param rclrVersion Version of rClr package. Default is 0.9.2 for Windows R4.
+#' @param developerVersion If `FALSE` (default), release verions of the packages
+#' will be installed. If `TRUE`, latest developer builds of the osps packages
+#' will be installed
+#' @param suppressOutput
+#'
+#' @return
+#' @export
+#'
+#' @examples
+installOSPPackages <- function(rtoolsPath = NULL, rclrVersion = "0.9.2",
+                               suppressOutput = TRUE,
```

Comment:
I would prefer the default setting `suppressOutput = FALSE,`

Reply:
Done.